recipes/startables/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "startables" %}
+{% set version = "0.8.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 186403164147fba3ee5e2e593290854645f29c7bf73cea5972107977a669774b
+
+build:
+  number: 0
+  skip: True  # [py<36]
```

Comment:
This recipe can be `noarch: python` if you remove this line.  The Python pin below takes care of the requirement in the selector here